aspnetcore/toc.yml

```diff
@@ -1,14 +1,8 @@
 items:
   - name: ASP.NET Core documentation
     href: index.yml
-  - name: Overview
-    items:
-      - name: About ASP.NET Core
-        uid: index
-      - name: Compare ASP.NET Core and ASP.NET
-        uid: fundamentals/choose-between-aspnet-and-aspnetcore
-      - name: Compare .NET and .NET Framework
-        href: /dotnet/standard/choosing-core-framework-server?toc=/aspnet/core/toc.json&bc=/aspnet/core/breadcrumb/toc.json
+  - name: About ASP.NET Core
```

Comment:
Did you want to reconcile the title, TOC entry, and file names?

Reply:
Sorry ... just saw your answer over there. It's getting late here ... east coast ⌚ ... getting sleepy 🛌💤.

> I'd like to keep the current Title (Overview of ASP.NET Core) and ToC entry (Overview) unchanged. I care less about the file name & URL, but if you want to update it to match the title, I'm fine with that. Maybe just overview.md? I assume we'll redirect the old URL to the new one.

Reply:
Yes, I'll add an entry to the redirection file.

> ToC entry (Overview)

... but the current entry is actually "About ASP.NET Core" ... so I think you're requesting that it be "Overview" instead.